docs/index.html

```diff
@@ -2015,14 +2015,531 @@ <h2>Vocabulary</h2>
+        },
+        "title": "Material the product is made from"
+      },
+      "https://schema.org/weight": {
+        "$comment": {
+          "term": "weight",
+          "@id": "https://schema.org/weight"
+        },
+        "title": "Weight of the product"
+      },
+      "https://schema.org/height": {
+        "$comment": {
+          "term": "height",
+          "@id": "https://schema.org/height"
+        },
+        "title": "Height of the product"
+      },
+      "https://schema.org/width": {
+        "$comment": {
+          "term": "width",
+          "@id": "https://schema.org/width"
+        },
+        "title": "Width of the product"
+      },
+      "https://schema.org/depth": {
+        "$comment": {
+          "term": "depth",
+          "@id": "https://schema.org/depth"
+        },
+        "title": "Depth of the Product"
+      },
+      "https://schema.org/url": {
+        "$comment": {
+          "term": "url",
+          "@id": "https://schema.org/url"
+        },
+        "title": "Url or the product item"
+      },
+      "https://schema.org/isBasedOn": {
+        "$comment": {
+          "term": "isBasedOn",
+          "@id": "https://schema.org/isBasedOn"
+        },
+        "title": "Is based on"
+      },
+      "https://schema.org/image": {
+        "$comment": {
+          "term": "image",
+          "@id": "https://schema.org/image"
+        },
+        "title": "Image of the product"
+      },
+      "https://schema.org/identifier": {
+        "$comment": {
+          "term": "HTScode",
+          "@id": "https://schema.org/identifier"
+        },
+        "title": "HTS code for product categorisation "
+      }
+    },
+    "examples": [
+      {
+        "@context": [
+          "https://w3id.org/traceability/v1"
+        ],
+        "type": "ProductRegistrationEvidenceDocument",
+        "category": "Non molestiae molestiae recusandae ut.",
+        "inProductGroupWithID": "Provident dolorem quis corporis eum nesciunt autem consequatur dignissimos.",
+        "productID": "6ce6d3d8-b0c3-473e-bfc6-6e2baa786fc5",
+        "mpn": "explicabo",
+        "gtin": "14075064217508",
+        "isAccessoryOrSparePartFor": "Veritatis dignissimos consequatur ab sed et labore consectetur nisi.",
+        "releaseDate": "11-2-2020",
+        "manufacturer": "non",
+        "globalLocationNumber": "6918901101918",
+        "leiCode": "Apt. 730",
+        "name": "aut",
+        "description": "Id mollitia quibusdam quia. Suscipit neque voluptates placeat. Iure et quia sit iusto rerum. Aliquid ut sed velit ut voluptas sit et debitis. Eos accusantium et.",
+        "model": "ca563607-4529-4d51-bc48-6023d5b5448a",
+        "color": "#580b67",
+        "material": "vitae",
+        "weight": "5.304 kg",
+        "height": "5.852 m",
+        "width": "8.141 m",
+        "depth": "3.787 m",
+        "url": "https://margaret.org",
+        "isBasedOn": "http://elwyn.net",
+        "image": "http://placeimg.com/640/480",
```

Comment:
unless existing are real and meant to stay that way
```suggestion
        "url": "https://margaret.example.org",
        "isBasedOn": "http://elwyn.example.net",
        "image": "http://placeimg.example.com/640/480",
```